ngraph/python/tests/test_ngraph/test_dft.py

```diff
@@ -0,0 +1,653 @@
+import ngraph as ng
+import numpy as np
+from tests import xfail_issue_49359
+from tests.runtime import get_runtime
+
+
+def build_fft_input_data():
+    input_data = [
+        0.85943836, 0.009941814, 0.004292889, 0.54598427, 0.8270831, 0.49770153, 0.9035636,
```

Comment:
We definitely need to make unit test input data smaller. 400 input elements looks as a overkill.

Reply:
May be we should check shapes only? We have single layer tests where we test output values. Here we should check that the operation can be instantiated. 

@akuporos what do you think about this?

Reply:
We can delete all other similar functions, if we will actively use `numpy.fft.fft`. The same can be done in `test_idft.py`.

Reply:
This amount of hardcoded input is an overkill to be sure. Is there any reason it can't be generated using random values?

Also the only-shape test would be nice. Outside of just confirming the result values.